android-sdk/src/main/java/ai/evolv/android_sdk/EvolvContextImpl.java

```diff
@@ -168,4 +189,242 @@ private void ensureInitialized() {
             }
         }
     }
+
+    @Override
+    public JsonElement get(String key) {
+        ensureInitialized();
+
+        if (key.equals("confirmations") || key.equals("contaminations")) {
```

Comment:
This check is to defend against people using the deprecated context storage locations. Please delete this check, and do not store the contaminations and confirmations only in the 'experiments.' namespace

Reply:
sure